### PR TITLE
Rename scripts/ directory to tools/

### PR DESCRIPTION
**Pull request by Holzhaus**
We also have a "script" directory (without the "s"), which makes it confusing
and harder to select the correct directory with shell completion.

The "tools" name fits better, since the directory contains tools that are/were
helpful during mixxx development (these tools are all scripts - but that
is of secondary importance and also coincidence).

### CODE REVIEW

**Comment by Be-ing** [comment]
Looks good to me. I'll wait to merge in case anyone objects.

File: tools/lp_to_github/community.txt [hunk 1]
```diff
@@ -0,0 +1,58 @@
+git bzr import lp:~ulatekh/mixxx/mingw
```
Comment (daschuer) [inline tools/lp_to_github/community.txt:1]: This one can be removed.

**Review by daschuer — CHANGES_REQUESTED** [review]
Please remove the lp_to_github files before the renaming, to keep this PR small and get rid of some unused legacy.

File: tools/lp_to_github/community.txt (hunk 1 above)
Comment (Holzhaus) [reply tools/lp_to_github/community.txt]: Done.

**Review by daschuer — CHANGES_REQUESTED** [review]
https://github.com/Holzhaus/mixxx/blob/f3a15cb1358ae9b06dc3f8bd057f239bc8a7acdd/tools/generate_sample_functions.py#L54

Needs to point to the new path.

And also 
src/engine/channelmixer_autogen.cpp:L6
src/test/enginebuffertest.cpp:L17
src/test/signalpathtest.h:L141
src/util/sample_autogen.h:L5

**Review by daschuer — APPROVED** [review]

**Comment by daschuer** [comment]
Thank you. LGTM.

**Comment by daschuer** [comment]
We may argue if this is really a 2.3 thing. 
But since it does not effect the binary, I go ahead and merge it. 
In future let's target refactoring to master.

**Pull request merged**